app/ClientApp/src/contexts/UserProvider.jsx

```diff
@@ -61,6 +61,16 @@ export const authApi = (user) => ({
       }
     }),
   post: (url, body, method = 'POST') =>
+    fetch(url, {
+      method,
+      headers: {
+        // eslint-disable-next-line no-underscore-dangle
+        Authorization: user.header,
+        'Content-type': 'application/json'
+      },
+      body
+    }),
+  delete: (url, body, method = 'DELETE') =>
```

Comment:
Since delete and other HTTP methods are used less often and closely resemble POST, the intention here was to optionally pass method to post method, so the intended way to call delete would be this:
```
authApi(user).post("https://<...>", JSON.parse({<...>}), "DELETE");
```

Reply:
Ah I guess that explains the method parameter xd I guess it doesn't really matter much, but should I leave it as is or change it?

Reply:
Leave as is